packages/hosted-widget-integration/src/utils/components/ManageInstrumentsModal/ManageInstrumentsModal.tsx

```diff
@@ -0,0 +1,252 @@
+import { PaymentInstrument } from '@bigcommerce/checkout-sdk';
+import { noop } from 'lodash';
+import React, { Component, ReactNode } from 'react';
+
+import { TranslatedString } from '@bigcommerce/checkout/locale';
+import { CheckoutContext } from '@bigcommerce/checkout/payment-integration-api';
+import { Button, ButtonSize, ButtonVariant, Modal, ModalHeader } from '@bigcommerce/checkout/ui';
+
+import { isAccountInstrument, isBankAccountInstrument, isCardInstrument } from '../../guards';
+import { ManageAccountInstrumentsTable } from '../ManageAccountInstrumentsTable';
+import { ManageCardInstrumentsTable } from '../ManageCardInstrumentsTable';
+import { ManageInstrumentsAlert } from '../ManageInstrumentsAlert';
+
+export interface ManageInstrumentsModalProps {
+    isOpen: boolean;
+    instruments: PaymentInstrument[];
+    onAfterOpen?(): void;
+    onDeleteInstrument?(instrumentId: string): void;
+    onDeleteInstrumentError?(error: Error): void;
+    onRequestClose?(): void;
+}
+
+export interface ManageInstrumentsModalState {
+    isConfirmingDelete: boolean;
+    selectedInstrumentId?: string;
+}
+
+// interface WithCheckoutProps {
+//     deleteInstrumentError?: Error;
+//     isDeletingInstrument: boolean;
+//     isLoadingInstruments: boolean;
+//     clearError(error: Error): Promise<CheckoutSelectors>;
+//     deleteInstrument(id: string): Promise<CheckoutSelectors>;
+// }
```

Comment:
Good catch, let me see what's going on here? Yes it's not needed anymore since we can access the values from context directly. I have removed the commented lines.